src/gui/osutils/winutils/WinUtils.cpp

```diff
@@ -18,13 +18,30 @@
 #include "WinUtils.h"
 #include <QAbstractNativeEventFilter>
 #include <QApplication>
+#include <QDebug>
 #include <QDir>
 #include <QSettings>
 
-#include <windows.h>
-
 QPointer<WinUtils> WinUtils::m_instance = nullptr;
 
+/**
+ *  Adds " [Screen Capture Allowed]" text to window title
+ * @param hwnd - handle of the window to update the title for
+ * @param title - current window title
+ * @return true if updated else false
+ */
+bool updateTitleWithSCAllowed(HWND hwnd, QString title) {
+    static const auto postfix = QObject::tr(" [Screen Capture Allowed]", "WinUtils");
+    if (not title.contains(postfix)) {
+        title.append(postfix);
+        if (SetWindowTextW(hwnd, (LPCWSTR)title.utf16())) {
+            return true;
+        }
+        qCritical().nospace() << "SetWindowTextW failed. error_code: 0x" << hex << GetLastError();
+    }
+    return false;
+}
+
```

Comment:
Remove this

Reply:
This code is part of error handling in the case `SetWindowDisplayAffinity` fails (user didn't specify `--allow-screencapture` in this case). It notifies the user via window title text that the window is no longer protected from screen capturing. We could also show error message box instead but I think it's more invasive, and to some extent less informative as you can forget that the screen protection is not enabled after you close the error message.

Reply:
I'd prefer to give the in-app warning instead.

Reply:
Ok, we could do via MessageWidget. In this case the issue can be modal popup dialogs (i.e. file save/open dialogs) as they could cover the warning and user might miss them. But yeah, in most cases it shouldn't come to this warning (I hope I'm not wrong :) so I think this solution is acceptable.

Reply:
I am totally unconcerned since the current state of the application has no restrictions.

Reply:
The error handling is now resolved via in app warning message.